### ISSUE TITLE
Integer overflow

### ISSUE THREAD

**Issue by schuemie** [OHDSI/Characterization#3]
When using larger cohort definitions IDs (e.g. those that are identical to the main concept ID of the cohort) I get 

```
java.sql.SQLException: [Amazon](500310) Invalid operation: Integer data overflow (multiplication) 
```

I'm pretty sure this originates from [here](https://github.com/OHDSI/Characterization/blob/main/inst/sql/sql_server/createTargetOutcomeCombinations.sql#L23). I think we don't need to construct cohort definition IDs like this in a temp table? Could just be a sequential number starting at 1?

**Comment by jreps** [comment]
Yeah - when I was writing this I realized the numbers would be an issue at some point but wanted to have a longer think about how to do the ids (which is why I added a comment on line 24 about using row ids instead). 

I think my current favorite is to create a table of target_id, outcome_id, type, row_id for all TnOs, Ts with NULL Os and Os with NULL T.  Then I can join onto this with T, O and type (or just T/O and type when there is no O/T) to get row_id as the cohort_id.
